.idea/.gitignore

```diff
@@ -0,0 +1,8 @@
+# Default ignored files
+/shelf/
```

Comment:
Please use .gitignore
https://mrkandreev.name/snippets/gitignore-generator/#WebStorm+all,VisualStudioCode
And you should open just your project folder. So these .idea folder and its files should be moved to corresponding folder.